libs/utils/trace.py

```diff
@@ -674,6 +676,19 @@ def _sanitize_CpuFrequency(self):
             return
         df = self._dfg_trace_event('cpu_frequency')
         clusters = self.platform['clusters']
+
```

Comment:
Can you put a comment here explaining where the 'cpu_frequency_devlib' events come from (I know it's obviously devlib but could mention the `trace_frequencies` method) and one below explaining the interleaving problem (as in the commit message)?